### PR TITLE
Button

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added the button components and its different variants

### CODE REVIEW

**Review by rahulk31 — COMMENTED** [review]
Looks great to me, Merge ready PR 👍

**Review by SalonSikdar — CHANGES_REQUESTED** [review]
The header for each button style can be a bit smaller. 
May I suggest adding a break after each set of header and components? 
```<header/>
<components.../>
<br/>
<header2/>
<components.../>
<br/>
```

You can put the entire components in an external container div with auto margin to center on the page. 
Great color scheme and, transforms.

File: component/button/button.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  color: var(--secondary-color);
+}
+
+.btn-link-accented {
+  color: var(--accent-color);
+}
+
+/** floating button **/
+
+.btn-float,
```
Comment (yaswanthmyneni) [inline component/button/button.css:72]: give box-shadow property for floating buttons.

File: component/button/button.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+.btn-outline-accented:hover,
+.btn-accented:hover,
+.btn-link-accented:hover {
+  background-color: var(--accent-hover-color);
+}
+
+.btn-link:hover {
+  text-decoration: underline;
+}
```
Comment (yaswanthmyneni) [inline component/button/button.css:118]: I didn't see its usage in preview.

File: component/button/button.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  padding: 0;
+  height: 3rem;
+  width: 3rem;
+  border-radius: 50%;
+  font-size: 1.2rem;
+}
+
+/** icon button **/
+
+.btn-only-icon {
```
Comment (yaswanthmyneni) [inline component/button/button.css:87]: I can remove it and add its width and height to the above class of the same name(btn-only-icon).

**Comment by nikhiljugale007** [comment]
Good to merge as per me.

**Comment by sumitk-singh** [comment]
Looks great to me,

File: component/button/button.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="button.css" />
+    <title>Document</title>
```
Comment (mehulsatardekar) [inline component/button/button.html]: Just a suggestion you can change the title of this page to **button** since it's a button page.

**Review by mehulsatardekar — COMMENTED** [review]
good to merge with the changes @Sahil25061999. Love that you have made isolated CSS classes for every element which gives more flexibility to users.

**Pull request merged**